apps-rendering/src/item.ts

```diff
@@ -387,6 +389,7 @@ const fromCapiLiveBlog =
 			blocks: parsedBlocks,
 			pagedBlocks,
 			totalBodyBlocks: content.blocks?.totalBodyBlocks ?? body.length,
+			pinnedPost: fromNullable(parsedBlocks.find((b) => b.isPinned)),
```

Comment:
What's the thinking behind storing `pinnedPost` in `Item`? So far I think it's only used in once place, in `LiveBlock`, so technically it could be derived directly from `blocks` there? Are there other advantages to having it derived earlier here?

Reply:
Very great point, as you said we only ever need to know about the pinned post within the `LiveBlocks` component so it makes sense to derive it in `LiveBlocks` rather than adding it to the `Item` model. Made the changes for this 👍